src/tests/ftest/harness/test_harness_tests.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from random import choice
+
+from apricot import TestWithServers
+from general_utils import run_task, pcmd
+
+
+class TestHarnessTests(TestWithServers):
+    # pylint: disable=too-few-public-methods
+    """Collection of tests to verify the behavior of the test harness.
+
+    :avocado: recursive
+    """
+
+    def test_core_files(self):
+        """Test to verify core file creation.
+
+        This test will send a signal 6 to a random daos_io_server process so
+        that it will create a core file, allowing the core file collection code
+        in launch.py to be tested.
+
+        :avocado: tags=medium,test_harness,core_files
+        """
+        # Choose a server find the pid of its daos_io_server process
+        host = choice(self.server_managers[0].hosts)
+        self.log.info("Obtaining pid of the daos_io_server process on %s", host)
+        task = run_task([host], "pgrep daos_io_server", 20)
```

Comment:
There can be more than one though, right?  Ahh.  Maybe that;s why you chose `medium`, temporarily, while you test this new code.

Reply:
The yaml specifies 4 hosts.

Reply:
But, on `medium,ib2` clusters, we run multiple IO servers, don't we?  Just saying that `buffer` could contain more than one `pid`, can't it?

Reply:
Not necessarily.  We only run tests that run multiple I/O servers on the `hw.medium,ib2` clusters, but that doesn't stop us from running other tests.  It is ultimately up to the test as to whether 1 or 2 I/O servers are run.

Since this test only starts one I/O server there should only be one `daos_io_server` process per host.  That said I've implemented a change in this space to display the output of `preg --list-full daos_io_server` and now use a regex to grab the pid.  As such it will only match the first pid it finds.